profab/ebs.py

```diff
@@ -22,14 +23,44 @@ def create(cls, server, size):
         """
         _logger.info("Creating volume size %dGB on %s",
             size, server.instance.placement)
-        volumes = server.get_volumes()
+        device = cls.get_volume(server.get_volumes())
+        volume = cls(server, server.cnx.create_volume(
+            size, server.instance.placement), device)
+        cls.attach_volume(server.insatance.id, volume, device)
+        return volume
+
+
+    @classmethod
+    def creat_from_snapshot(cls, server, snapshot):
+        """Create a new volume from snapshot.
+        """
+        _logger.info("Creating volume size 200GB from snapshot id %s",
```

Comment:
The message says that this is always a 200GB volume -- surely it is whatever size the snapshot is? I think we should either leave out the size from the log message, or work out what the correct size is to put in here.